Detectors/MUON/MCH/PreClustering/CMakeLists.txt

```diff
@@ -18,27 +16,16 @@ set(LIBRARY_NAME ${MODULE_NAME})
 set(BUCKET_NAME mch_preclustering_bucket)
 
 # Define application
-Set(Exe_Names
-digit-dumper
-preclusterizer
-)
+O2_GENERATE_EXECUTABLE(
+  EXE_NAME preclusterizer-workflow
```

Comment:
It's true we don't seem to currently have a naming convention for executables. Still, knowing that all binaries end up in the same installation bin/ directory, I would prefix this one by `mch-`